rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -604,6 +604,45 @@ def mock_check_asset_existence(identifier: str, query_packaged_db: bool = True)
         'Try redecoding the event(s) or check the logs for more details.',
     ]
 
+    # check that address filter shows incoming transactions to the filtered address.
+    with rotki.data.db.conn.write_ctx() as write_cursor:
+        DBHistoryEvents(rotki.data.db).add_history_events(
+            write_cursor=write_cursor,
+            history=[EvmEvent(
+                tx_hash=deserialize_evm_tx_hash('0x9a76e51e6feb83690b4f0ecb257adbceb73b6f8b38d7d5c5d3f5e22fd10e3c71'),
+                sequence_index=100,
+                timestamp=TimestampMS(1639924590000),
+                location=Location.ETHEREUM,
+                event_type=HistoryEventType.RECEIVE,
+                asset=A_ETH,
+                amount=FVal('2.5'),
+                location_label=(address := string_to_evm_address('0xA7C8F1e13eDC5FBfB768f55ECF2Fee5d4C5BF964')),  # noqa: E501
+                notes='Receive 2.5 ETH from 0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12',
+                event_subtype=HistoryEventSubType.NONE,
+                address=string_to_evm_address('0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12'),
+            ), EvmEvent(
+                tx_hash=deserialize_evm_tx_hash('0x9a76e51e6feb83690b4f0ecb257adbceb73b6f8b38d7d5c5d3f5e22fd10e3c72'),
+                sequence_index=10,
+                timestamp=TimestampMS(1639924990000),
+                location=Location.ETHEREUM,
+                event_type=HistoryEventType.SPEND,
```

Comment:
And generally adjust to make sure that this entire test matches a transfer between tracked addresses

```suggestion
                event_type=HistoryEventType.TRANSFER,
```